### DISCUSSION TITLE
Обговорення питань для JavaScript-інтерв'ю. 📒

### DISCUSSION THREAD

**Discussion by ViktorSvertoka** [FrontendLovers/javascript-interview-questions#1] [Q&A] [answered]
Цей розділ призначений для обговорення питань, пов'язаних з JavaScript-інтерв'ю.

**Comment by Tyreee925** [comment, marked as answer]
Вітаю! Гарна робота! Є лише одне зауваження: якщо додається запитання про ECMA, можливо, варто було б спершу ознайомитися зі специфікацією, аби уникнути поширення міфів чи неточностей про цю мову програмування.

**Reply by ViktorSvertoka** [reply]
Вітаю! Дякую! Буду мати на увазі.
